action/index.js

```diff
@@ -0,0 +1,50 @@
+const core = require('@actions/core');
+const github = require('@actions/github');
+const child_process = require('child_process');
+const fs = require('fs');
+
+/**
+ * Dump contents of file to console.
+ * @param {string} f - file to dump
+ */
+function dump(f) {
+  if (!fs.existsSync(f)) {
+    console.log(`${f} doesn't exist, skipping`);
+    return;
+  }
+  const data = fs.readFileSync(f, 'utf8');
+  console.log(`${f}: ${data}`);
```

Comment:
I'd just:

```js
try {
  const data = fs.readFileSync(path, 'utf8');
  console.log(`${path}: ${data}`);
} catch (e) {
  console.log(`${path} doesn't exist, skipping`);
}
```

Which might catch some other errors.
The empty return looks a bit weird in JS; an if-else is more common.